src/functions.php

```diff
@@ -54,11 +52,9 @@ function str(MessageInterface $message)
  *
  * @param string|UriInterface $uri
  *
- * @return UriInterface
- *
  * @throws \InvalidArgumentException
  */
-function uri_for($uri)
+function uri_for($uri): UriInterface
```

Comment:
Since we are here.. maybe we could also fix the docblock of this function by adding the leading slash

`This function accepts a string or {@see \Psr\Http\Message\UriInterface} and`